crates/net/network/src/discovery.rs

```diff
@@ -43,23 +43,25 @@ impl Discovery {
     pub async fn new(
         discovery_addr: SocketAddr,
         sk: SecretKey,
-        dsicv4_config: Discv4Config,
+        discv4_config: Discv4Config,
```

Comment:
I feel like this should be an option, same as the update stream.

So no new to open a socket and do nothing with it.